src/gui/guiSkin.cpp

```diff
@@ -24,95 +24,47 @@ GUISkin::GUISkin(EGUI_SKIN_TYPE type, video::IVideoDriver* driver)
-		Colors[EGDC_ICON]               = video::SColor(200,255,255,255);
-		Colors[EGDC_ICON_HIGH_LIGHT]    = video::SColor(200,8,36,107);
-		Colors[EGDC_GRAY_WINDOW_SYMBOL] = video::SColor(240,100,100,100);
-		Colors[EGDC_EDITABLE] 			= video::SColor(255,255,255,255);
-		Colors[EGDC_GRAY_EDITABLE]		= video::SColor(255,120,120,120);
-		Colors[EGDC_FOCUSED_EDITABLE]	= video::SColor(255,240,240,255);
-
-
-		Sizes[EGDS_SCROLLBAR_SIZE] = 14;
-		Sizes[EGDS_MENU_HEIGHT] = 30;
-		Sizes[EGDS_WINDOW_BUTTON_WIDTH] = 15;
-		Sizes[EGDS_CHECK_BOX_WIDTH] = 18;
-		Sizes[EGDS_MESSAGE_BOX_WIDTH] = 500;
-		Sizes[EGDS_MESSAGE_BOX_HEIGHT] = 200;
-		Sizes[EGDS_BUTTON_WIDTH] = 80;
-		Sizes[EGDS_BUTTON_HEIGHT] = 30;
-
-		Sizes[EGDS_TEXT_DISTANCE_X] = 2;
-		Sizes[EGDS_TEXT_DISTANCE_Y] = 0;
-
-		Sizes[EGDS_TITLEBARTEXT_DISTANCE_X] = 2;
-		Sizes[EGDS_TITLEBARTEXT_DISTANCE_Y] = 0;
-	}
-	else
-	{
-		//0x80a6a8af
-		Colors[EGDC_3D_DARK_SHADOW] 	=	0x60767982;
-		//Colors[EGDC_3D_FACE]			=	0xc0c9ccd4;		// tab background
-		Colors[EGDC_3D_FACE]			=	0xc0cbd2d9;		// tab background
-		Colors[EGDC_3D_SHADOW]			=	0x50e4e8f1;		// tab background, and left-top highlight
-		Colors[EGDC_3D_HIGH_LIGHT]		=	0x40c7ccdc;
-		Colors[EGDC_3D_LIGHT]			=	0x802e313a;
-		Colors[EGDC_ACTIVE_BORDER]		=	0x80404040;		// window title
-		Colors[EGDC_ACTIVE_CAPTION] 	=	0xffd0d0d0;
-		Colors[EGDC_APP_WORKSPACE]		=	0xc0646464;		// unused
-		Colors[EGDC_BUTTON_TEXT]		=	0xd0161616;
-		Colors[EGDC_GRAY_TEXT]			=	0x3c141414;
-		Colors[EGDC_HIGH_LIGHT]			=	0x6c606060;
-		Colors[EGDC_HIGH_LIGHT_TEXT]	=	0xd0e0e0e0;
-		Colors[EGDC_INACTIVE_BORDER]	=	0xf0a5a5a5;
-		Colors[EGDC_INACTIVE_CAPTION]	=	0xffd2d2d2;
-		Colors[EGDC_TOOLTIP]			=	0xf00f2033;
-		Colors[EGDC_TOOLTIP_BACKGROUND]	= 	0xc0cbd2d9;
-		Colors[EGDC_SCROLLBAR]			= 	0xf0e0e0e0;
-		Colors[EGDC_WINDOW]				= 	0xf0f0f0f0;
-		Colors[EGDC_WINDOW_SYMBOL]		= 	0xd0161616;
-		Colors[EGDC_ICON]				= 	0xd0161616;
-		Colors[EGDC_ICON_HIGH_LIGHT]	= 	0xd0606060;
-		Colors[EGDC_GRAY_WINDOW_SYMBOL] = 	0x3c101010;
-		Colors[EGDC_EDITABLE] 			= 	0xf0ffffff;
-		Colors[EGDC_GRAY_EDITABLE]		= 	0xf0cccccc;
-		Colors[EGDC_FOCUSED_EDITABLE]	= 	0xf0fffff0;
-
-		Sizes[EGDS_SCROLLBAR_SIZE] = 14;
-		Sizes[EGDS_MENU_HEIGHT] = 48;
-		Sizes[EGDS_WINDOW_BUTTON_WIDTH] = 15;
-		Sizes[EGDS_CHECK_BOX_WIDTH] = 18;
-		Sizes[EGDS_MESSAGE_BOX_WIDTH] = 500;
-		Sizes[EGDS_MESSAGE_BOX_HEIGHT] = 200;
-		Sizes[EGDS_BUTTON_WIDTH] = 80;
-		Sizes[EGDS_BUTTON_HEIGHT] = 30;
-
-		Sizes[EGDS_TEXT_DISTANCE_X] = 3;
-		Sizes[EGDS_TEXT_DISTANCE_Y] = 2;
-
-		Sizes[EGDS_TITLEBARTEXT_DISTANCE_X] = 3;
-		Sizes[EGDS_TITLEBARTEXT_DISTANCE_Y] = 2;
-	}
+	Colors[EGDC_3D_DARK_SHADOW]     = video::SColor(101,50,50,50);
+	Colors[EGDC_3D_SHADOW]          = video::SColor(101,130,130,130);
+	Colors[EGDC_3D_FACE]            = video::SColor(220,70,70,70);
+	Colors[EGDC_3D_HIGH_LIGHT]      = video::SColor(101,255,255,255);
+	Colors[EGDC_3D_LIGHT]           = video::SColor(101,210,210,210);

```

Comment:
the faux 3d thing should probably be disabled too?

Reply:
@rollerozxa

Reply:
shouldn't that be kept such that the buttons still look "clickable"?

Reply:
dunno, a comparison image would help judging this I think

Reply:
@rollerozxa reminder